sunpy/conftest.py

```diff
@@ -47,17 +47,10 @@ def pytest_runtest_setup(item):
 
 
 def pytest_unconfigure(config):
-    if len(figure_test_pngfiles) > 0:
-        tempdir = tempfile.mkdtemp(suffix="_figures")
-
-        # Rename each PNG with the name of the corresponding test
-        for test_name in figure_test_pngfiles:
-            os.rename(figure_test_pngfiles[test_name], os.path.join(tempdir, test_name + '.png'))
-
+    if len(new_hash_library) > 0:
         # Write the new hash library in JSON
-        hashfile = os.path.join(tempdir, HASH_LIBRARY_NAME)
+        hashfile = os.path.join(test_fig_dir, HASH_LIBRARY_NAME)
         with open(hashfile, 'w') as outfile:
             json.dump(new_hash_library, outfile, sort_keys=True, indent=4, separators=(',', ': '))
 
-        print('All test files for figure hashes can be found in {0}'.format(tempdir))
```

Comment:
Do we not want this print statement somewhere else instead?

Reply:
I've put a print statement back in.